### ISSUE TITLE
python-jose is not compatible with python 3.6

### ISSUE THREAD

**Issue by hackalyze** [spulec/moto#1712]
Recently the python-jose requirement was added to moto in PR https://github.com/spulec/moto/pull/1617, however python-jose breaks compatibility with python 3.6.

See a similar issue that was raised in warrant: https://github.com/capless/warrant/issues/31

As mentioned in the above issue, pycryptodome is the preferred solution: https://github.com/Legrandin/pycryptodome

When I try to pipenv install git+git://github.com/spulec/moto.git#egg=moto I get the following in python 3.6:

Error:  An error occurred while installing git+git://github.com/spulec/moto.git#egg=moto!
  Could not find a version that satisfies the requirement python-jose<3.0.0 (from moto) (from versions: )
No matching distribution found for python-jose<3.0.0 (from moto)

**Comment by jbergknoff-rival** [comment]
python-jose<3.0.0 doesn't break compatibility with python 3.6. There must be something else going on in your setup, the blank list of versions is especially strange. Maybe you're pointing at a non-PyPi registry?

```
$ docker run -it --rm python:3.6.6-alpine3.8 sh
# pip install "python-jose<3.0.0"
Collecting python-jose<3.0.0
  Downloading https://files.pythonhosted.org/packages/bf/5c/5fa238c0c5b0656994b52721dd8b1d7bf52ebd8786518dde794f44de86b6/python_jose-2.0.2-py2.py3-none-any.whl
Collecting ecdsa<1.0 (from python-jose<3.0.0)
  Downloading https://files.pythonhosted.org/packages/63/f4/73669d51825516ce8c43b816c0a6b64cd6eb71d08b99820c00792cb42222/ecdsa-0.13-py2.py3-none-any.whl (86kB)
    100% |████████████████████████████████| 92kB 3.9MB/s 
...
```

**Comment by bblommers** [comment]
This should now be fixed by #3049, and available as of moto >= 1.3.15.dev886

**Comment by MartinFalatic** [comment]
I'm eager to see this in a point release, as it should resolve a few unnecessary dependencies, particularly on `rsa`. Is there an ETA on the next moto release?
